statistics/fmsketch.go

```diff
@@ -96,6 +96,24 @@ func (s *FMSketch) InsertValue(sc *stmtctx.StatementContext, value types.Datum)
 	return nil
 }
 
+// InsertRowValue inserts multi-column values to the sketch.
+func (s *FMSketch) InsertRowValue(sc *stmtctx.StatementContext, values []types.Datum) error {
+	b := make([]byte, 0, 8)
+	s.hashFunc.Reset()
+	for _, v := range values {
+		b, err := codec.EncodeValue(sc, b, v)
+		if err != nil {
+			return err
+		}
+		_, err = s.hashFunc.Write(b)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
Would it be more efficient to call `s.hashFunc.Write()` only once after encoding the while row?